lib/plugins/aws/provider/awsProvider.js

```diff
@@ -194,6 +194,13 @@ class AwsProvider {
     impl.addEnvironmentProfile(result, `AWS_${stageUpper}`);
 
     result.region = this.getRegion();
+
+    const deploymentBucketObject = this.serverless.service.provider.deploymentBucketObject;
+    if (deploymentBucketObject && deploymentBucketObject.serverSideEncryption
```

Comment:
Thank you very much for the clarification @jayzalowitz and @Athulus 👍 

That makes sense.